benchmark/net/chunky_http_client.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+// test HTTP throughput in fragmented header case
+var common = require('../common.js');
+var net = require('net');
+var test = require('../../test/common.js');
+
+var bench = common.createBenchmark(main, {
+  len:  [1, 4, 8, 16, 32, 64, 128],
+  num:  [5, 50, 500, 2000],
+  type: ['send'],
+});
+
+
+function main(conf) {
+  var len = +conf.len;
+  var num = +conf.num;
```

Comment:
Are you sure `conf.len` and `conf.num` will always be defined? If not they'll return `NaN`.